virtcontainers/store/filesystem_backend.go

```diff
@@ -186,6 +186,7 @@ func (f *filesystem) new(ctx context.Context, path string, host string) error {
 }
 
 func (f *filesystem) delete() error {
+	f.logger().Debugf("Deleting files %s", f.path)
```

Comment:
I think something like this is better

`f.logger().WithField("path", f.path).Debugf("Deleting files)`

Reply:
Thanks @devimc !